src/components/text_field/TextField.vue

```diff
@@ -16,6 +16,7 @@
       :type='inputType'
       :value='value'
       @input='updateValue'
+      @change="handleChange"
```

Comment:
Just set change event on input element, and the textarea element need this event too.

Reply:
Hello, thanks to you, sorry, I goint to add it and to do an other pull. And I going to use yarn too for dependencies.